core/server/models/base/index.js

```diff
@@ -459,7 +462,7 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
 
         return model.fetch(options).then(function then(object) {
             if (object) {
-                return object.save(data, options);
+                return object.save(data, _.merge({method: 'update'}, options));
```

Comment:
In bookshelf there are three methods: `update`, `patch` and `insert`.
Based on this type, we and Bookshelf can differentiate what todo. If you update a post model anywhere in Ghost (`Post.edit`), we ensure that we forward the method property. See this example usages: https://github.com/TryGhost/Ghost/pull/8328/files#diff-eb942d2a7406b236df44c35a83722627R33 or https://github.com/TryGhost/Ghost/pull/8328/files#diff-e05bf798b07f40dd2e22a828f86eee13R273